README.md

```diff
@@ -290,6 +291,7 @@ You can provide settings to a Hops application via a `"hops"` key in your `packa
 | `serverDir` | `String` | `node_modules/.cache/untool` | `<rootDir>/dist` | The directory where the generated server middleware will be stored |
 | `browsers` | `Array<String>` | `['defaults']` | `['last 1 Chrome versions']` | An array of browserslist queries to specify targets for which to transpile/polyfill (see [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env) for more information) |
 | `node` | `String` | `current` | `8.10` | A Node.js version identifier or `current` to specify for which target to transpile/polyfill |
+| `browserWhitelist` | `Object` | `{"basePath":true}` | A map of config keys that should be whitelisted and exposed to the client. Nested paths can be described with dot notation |
```

Comment:
Mentioning that those config keys are whitelisted, is meaningless in this context imho.

```diff
- A map of config keys that should be whitelisted and exposed to the client.
+ A map of config keys that should be exposed to the client.
```

`Using` sounds a little better than `with` here imho.

```diff
- Nested paths can be described with dot notation
+ Nested paths can be described using dot notation
```